build/Foundation/dll/Foundation.def

```diff
@@ -6,6 +6,7 @@ LIBRARY Foundation
 
         ; Foundation ObjC classes
         _OBJC_CLASS_NSArray DATA
+        _OBJC_CLASS__NSProxyOrderedSetArray DATA
```

Comment:
What was the error? I can't think of a reason why it would be necessary to export this class.

---
In reply to: [64045867](https://github.com/Microsoft/WinObjC/pull/482#discussion_r64045867) [](ancestors = 64045867)